google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -352,8 +385,8 @@ def submit(
         if service_account:
             self._gca_resource.service_account = service_account
 
-        if network:
-            self._gca_resource.network = network
+        if network is not None:
+            self._gca_resource.network = network or initializer.global_config.network
```

Comment:
This seems to not set `network` from `initializer.gloabel_config` by default.

Reply:
Fixed, set `network` argument beforehand and check if `network` arg has been set in the if statement